xbmc/addons/addoninfo/AddonType.h

```diff
@@ -71,6 +71,14 @@ typedef enum
   ADDON_MAX
 } TYPE;
 
+/* \brief Addon types associated to dependency addons
+ */
+static const std::set<TYPE> dependencyTypes = {
```

Comment:
Here you moved back to *.h, thought before to have on *.cpp as:
```cpp
namespace ADDON
{
static const std::set<TYPE> dependencyTypes = {
    ADDON_SCRAPER_LIBRARY,
    ADDON_SCRIPT_LIBRARY,
    ADDON_SCRIPT_MODULE,
};
} /* namespace ADDON */
```

His `using namespace ADDON;` on begin of cpp not switch them to it, thats why I comes to it before.